script-with-argparse.py

```diff
@@ -0,0 +1,223 @@
+            sent_3 = f"The domain is: {self.domain}."
+        else:
+            sent_3 = "The domain is unspecified." # if no domain was added, the text mentions it
+        sent_4 = f"It's {self.words} words long, with a rate of {round(self.rate, 2)} € per word." #the word rate is rounded to two decimal places to avoid cumbersomely long numbers
+        # this if-statement considers whether the deadline is in the past
+        if self.dl < datetime.date.today():
+            sent_5 = f"It started on {self.st} and was due on {self.dl}, so {self.length.days} days were foreseen for it. To meet the deadline, {round(self.efficiency)} words needed to be translated or revised per day." # the efficiency is rounded to units because you can't translate a fraction of a word anyway
+        else:
+            sent_5 = f"It started on {self.st} and is due on {self.dl}, so {self.length.days} days are foreseen for it, of which {self.daysleft.days} left. To meet the deadline, {round(self.efficiency)} words need to be translated or revised per day."
+        # this if-statement considers whether there is a translation memory for the project
+        sent_6 = f"There is {'a' if self.tm else 'no'} translation memory."
+        sent_7 = f"The project is currently {self.status}."
+        # print each sentence in a different line
+        return "\n".join([sent_1, sent_2, sent_3, sent_4, sent_5, sent_6, sent_7])
+
+if __name__ == "__main":
+    parser = argparse.ArgumentParser("project")
+    parser.add_argument("title", type=str, help="Title of the project")
+    parser.add_argument("client", type=str, help="Client who ordered the translation")
+    parser.add_argument("source", type=str, help="Source language")
+    parser.add_argument("target", type=str, help="Target language")
+    parser.add_argument("words", type=int, help="Project word count")
+    parser.add_argument("start", type=str, help="Project start date")
+    parser.add_argument("deadline", type=str, help="Project deadline")
+    parser.add_argument("price", type=float, help="Total price project")
+    parser.add_argument("tm", type=bool, help="Whether a translation memory is available")
+    parser.add_argument("--translator", type=str, default = "internal", help="Translator assigned to the project")
+    parser.add_argument("--revisor", type=str, default = "internal",help="Revisor assigned to the project")
+    parser.add_argument("--status", type=str, default = "created",
+                            choices = ["created", "in translation", "in revision", "delivered", "delayed", "cancelled", "canceled"], help="Project status in the agency workflow")
+    parser.add_argument("--domain", type=str, default = "", help="Overall domain of the text")
+    args = parser.parse_args()
+    proj = Project(args.title, args.client, args.source, args.target, args.words, args.start, args.deadline, args.price, args.tm, args.translator, args.revisor, args.status, args.domain)
+    proj.days_left()
```

Comment:
you will need to explicitly `print()` your variables for them to be visible when running the script.

Reply:
I tried to add it, but I'm not sure I've done it correctly. Could you have a look? Many thanks in advance!

Reply:
Just adding `print()` like you did in line 229 should work (also for anything else you have to print.
You can test it by running the script! Sorry it was never going to work with the wrong "__main" above...